Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,52 @@ label monika_literature_value:
+    
+# Do you care about the environment?
+default persistent._mas_pm_cares_environment = None
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_environment",category=['nature', 'life'],prompt="The environment",random=True))
+ 
+label monika_environment:
+    m 2esd "Hey [player], I've been wondering..."
+    $ _history_list.pop()
+    menu:
+        m "Do you care about the environment{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_cares_environment = True
+            m 2suo "Really? That's wonderful!"
+            m 2wsa "I'm so glad to hear that, [player]."
+            m 3esd "Personally, I think that the environment is something we should treasure."
+            m 3esa "It's a beautiful world filled with all kinds of life, and there's a lot nature does to lend humanity a hand."
+            m 4esb "Insects pollinate crops, decomposers recycle waste, trees provide food and shade, oceans absorb heat, and so on."
+            m 1esu "Since nature does so much to help us, I think it's only fair that we give something back in return."
+            m 7hub "So, here's Monika's Green Tip of the Day!"
+            m 7esc "Sometimes, it might seem difficult to make a difference when products such as electric vehicles and solar roofs can be expensive."
+            m 3eub "However, you can make a difference and save money just by performing a few simple tasks each day!"
+            m 3esb "Turning off appliances, taking shorter showers, buying a reusable water bottle, commuting by public transport..."
+            m 4esa "If you feel up to it, you can even become a vegetarian and start growing your own garden!"
+            m 7eud "Voting and engaging in your local community can go a long way as well; if you take the initiative, others are sure to follow!"
+            m 3esu "The important thing is to work hard and stay optimistic. If you do that, there's nothing you can't accomplish."
+            m 3wud "Who knows, maybe going green will even make you healthier and happier."
+            m 4esu "After all, a sustaininable life is a satisfying life."
+            m 3esa "For me, at least."
+            m 3esb "That's my advice for today!"
+            m 2hka "Thanks for listening~"
+     
+        "No.":
+            $ persistent._mas_pm_cares_environment = False
+            m 1duc "Hmm..."
+            m 3lsd "Well, I can see why you may feel that way; natural can be cruel."
+            m 4ekc "Diseases and natural disasters devastate millions, pests ravage crops, plant roots can damage infrastructure, and some insects are flat out obnoxious."
+            m 2ekx "I mean, I don't think I've ever met anyone who likes mosquitos."
+            m 2esd "There's also the fact that some people enjoy a comfortable standard of living."
+            m 3euc "Thanks to technology, we live in a more convienent and connected world than ever, and sometimes it may feel like nature does nothing but get in the way."
```

Comment:
```suggestion
            m 3euc "Thanks to technology, we live in a more convenient and connected world than ever, and sometimes it may feel like nature does nothing but get in the way."
```